_posts/en/newsletters/2019-10-09-newsletter.md

```diff
@@ -0,0 +1,171 @@
+
+- [LND #3545][] adds [documentation][lnd repo doc] explaining how to use
+  LND's new reproducible build system.  This should allow anyone with
+  moderate technical skills to build identical binaries to those
+  released by Lightning Labs, ensuring that users are running the
+  peer-reviewed code from the LND repository and its dependencies.
+
+- [LND #3365][] adds support for using `option_static_remotekey`
+  commitment outputs as described [later in this section][opt static
+  remotekey].  LND calls this new feature either "tweakless commitments"
+  (commitments without any pubkey tweaking) or "safu commitments"
+  (Situation All Fucked Up commitments).  As that second name implies,
+  this new commitment protocol particularly shines when something has
+  gone wrong and you've lost data.  If that happens, you need only wait
+  for your channel counterparty to close the channel by paying a key
+  directly derived from your HD wallet.  Because the key was generated
+  without any additional data ("tweaking"), your wallet doesn't need any
+  extra data in order to find and spend your funds.  This is a simplified
+  alternative to the [data loss protection][] protocol that LND
+  previously used and continues to understand.
+
+- [C-Lightning #3078][] adds support for creating and using channels
+  that spend Liquid-BTC on the Liquid sidechain.
+
+- [C-Lightning #2803][] adds a new python package named `pyln` that
+  includes a partial implementation of the LN specification.  As
+  described in its [documentation][pyln-proto readme], "This package
+  implements some of the Lightning Network protocol in pure python. It
+  is intended for protocol testing and some minor tooling only. It is
+  not deemed secure enough to handle any amount of real funds (you have
+  been warned!)."
+
+- [C-Lightning #3062][] causes the `plugin` command to return an error if a
+  requested plugin hasn't reported successful startup within 20 seconds.
+
+- [BOLTs #676][] amends [BOLT2][] to specify that a node should not send
+  the `funding_locked` message until it has validated the funding
+  transaction.  This warns future implementers about the problem that
+  lead to the vulnerabilities [described in last week's newsletter][ln
+  vuln disclosure].
+
+- [BOLTs #642][] allows two peers opening a channel to negotiate an
+  `option_static_remotekey` flag.  If both peers set this flag, any
+  commitment transactions they create which they're able to spend
+  unilaterally (e.g. to force close the channel) must pay their peer's
+  funds to a static address negotiated during the initial channel open.
+  For example, if Alice and Bob both request `option_static_remotekey`,
+  any commitment transactions that Alice can publish onchain must pay
+  bc1bob[...] and any commitment transactions that Bob can publish
+  onchain must pay bc1ally[...].  If at least one of them doesn't set
+  this flag, they'll fallback to the older protocol of using a different
```

Comment:
Unsure but iirc "fallback" is a noun and "fall back" is the verb, e.g. "we'll fall back to the fallback plan", similar to "we'll set up the setup"